source/Octo.Tests/Commands/VersionTestFixture.cs

```diff
@@ -0,0 +1,55 @@
+using System;
+using System.IO;
+using FluentAssertions;
+using NUnit.Framework;
+using Octopus.Cli.Commands;
+using Octopus.Cli.Tests.Helpers;
+using Octopus.Cli.Util;
+using Serilog;
+
+namespace Octopus.Cli.Tests.Commands
+{
+    [TestFixture]
+    public class VersionTestFixture
+    {
+        VersionCommand versionCommand;
+        StringWriter output;
+        TextWriter originalOutput;
+        private ICommandOutputProvider commandOutputProvider;
+        private ILogger logger;
+
+
+        [SetUp]
+        public void SetUp()
+        {
+            originalOutput = Console.Out;
+            output = new StringWriter();
+            Console.SetOut(output);
+
+            commandOutputProvider = new CommandOutputProvider(logger);
+            versionCommand = new VersionCommand(commandOutputProvider);
+            logger = new LoggerConfiguration().WriteTo.TextWriter(output).CreateLogger();
+        }
+
+        [Test]
+        public void ShouldPrintCorrectVersionNumber()
+        {
+            var version = GetVersionFromFile(Path.Combine(TestContext.CurrentContext.WorkDirectory, "ExpectedSdkVersion.txt"));
```

Comment:
ExpectedSdkVersion.txt needs to be added to .gitignore as it is not removed after a test, but should not be included in the repo. If it is not excluded and IDE will incorrectly automatically include it at some point.